base/applications/mshta/main.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * PROJECT:         HTML Application Viewer
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * FILE:            base/applications/mshta/main.c
+ * PURPOSE:         HTML Application Host
+ * PROGRAMMERS:     Jared Smudde (computerwhiz02@hotmail.com)
+ */
```

Comment:
Hi, can you please use the new file headers as shown in https://www.reactos.org/wiki/Coding_Style#File_Structure ? (please respect in there also the indentation/spacing).